tox.ini

```diff
@@ -1,15 +1,14 @@
 
 [tox]
-minversion = 1.8
+minversion = 3.5.3
```

Comment:
Do we **require** the latest version?

For testing inside Travis this is fine, though for contributors running the tests locally is their any reason they couldn't such as Python 2 users?

Reply:
1.8 is way too outdated (4 years old). It's safer to rely on improvements/bugfixes with this.
From my experience, tox has always had lots of unpleasant issues and I'd never force users to go through those levels of hell when something breaks, it's impossible to debug and when you finally dig into it turns out it's been fixed or so. A friendly message "your tox is outdated, please upgrade" is way better.
wrt py2, 2.7 is still supported, 2.6 is not needed as Molecule never declared its support nor attempted to have code/tests compatible with it.
FWIW even having tox installed under Python 3 doesn't mean user is locked to using it. tox creates virtualenvs based on different pythons regardless.
P.S. If users want conflicting deps they must use virtualenvs.